README.md

```diff
@@ -21,6 +21,7 @@ Prerequisites:
 * [DependencyInjection](src/DependencyInjection) - How to inject dependencies in activities and use generic hosts for workers
 * [Encryption](src/Encryption) - End-to-end encryption with Temporal payload codecs.
 * [Mutex](src/Mutex) - How to implement a mutex as a workflow. Demonstrates how to avoid race conditions or parallel mutually exclusive operations on the same resource.
+* [OpenTelemetry](src/OpenTelemetry) - Demonstrates how to set up OpenTelemetry tracing and metrics for both the client and worker, using both the .NET metrics API and internal forwarding from the Core SDK.
```

Comment:
If you are going to keep these as separate projects, may want a `src/OpenTelemetry/README.md` that shows the two subdir projects (granted I don't think we need entirely separate projects for something as simple as switching the metric config).

Reply:
Fixed